History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
```

Comment:
I'd say, let's do this in a separate PR. `observe` and `observeChanges` do have separate `this` so jus adding a context to `bindEnvironment` isn't going to set the context for both `observe` and `observeChanges`.